arlo-client/src/components/Header.test.tsx

```diff
@@ -92,4 +102,30 @@ describe('Header', () => {
       expect(loginButton).toBeFalsy()
     })
   })
+
+  it('shows the nav bar when authenticated and there is an electionId', async () => {
+    checkAndToastMock.mockReturnValue(false)
+    apiMock.mockImplementation(async () => ({
```

Comment:
any way to make this mock respond specifically to `/auth/me` or to check that that's what it was called with?

Reply:
@benadida We can test for that, sure. It's being called from `<AuthDataProvider>` though, and that's tested in that respect in multiple other places, and isn't really related to the `Header` component specifically. That's why I left if off here. Thoughts?

Reply:
the only thing I want to make sure is that the code isn't calling some other API that's not /auth/me, say there's a typo and it's calling `/aut/me`, would the mock here still respond successfully?

Reply:
@benadida Ah, yes, that is covered. This one here in particular would still respond successfully, but the code that's actually calling `/auth/me` is in the `<AuthDataProvider>` component, and if that got changed then it would fail a bunch of focused tests on that. :slightly_smiling_face:

Reply:
Got it, so the Header code is `not` explicitly calling an API, it's referencing another piece of code where that's tested. Good for me then.